### PR TITLE
 Fix course permissions and required parameters in grader plugin

### PR DESCRIPTION
**Pull request by crdgonzalezca**
This PR is fixing some course permissions in the api pages and validating mandatory parameters.

This is taking new changes made in utils PR.

### CODE REVIEW

File: inginious/frontend/plugins/grader_generator/__init__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 def init(plugin_manager, course_factory, client, config):
     plugin_manager.add_page(r'/grader_generator/static/(.*)', create_static_resource_page(_BASE_STATIC_FOLDER))
 
-    plugin_manager.add_page('/api/stats/test_file_api', TaskTestCasesFilesApi)
+    plugin_manager.add_page('/api/grader_generator/test_file_api', TaskTestCasesFilesApi)
```
Comment (amrondonp) [inline inginious/frontend/plugins/grader_generator/__init__.py:16]: Good catch 👍👍

File: inginious/frontend/plugins/grader_generator/static/js/grader_generator.js [hunk 2]
```diff
@@ -117,7 +117,7 @@ function studio_set_initial_problem(initialProblemId){
 
 function studio_update_grader_files()
 {
-    $.get('/api/stats/test_file_api', {
+    $.get('/api/grader_generator/test_file_api', {
```
Comment (amrondonp) [inline inginious/frontend/plugins/grader_generator/static/js/grader_generator.js:120]: 👍

**Review by amrondonp — COMMENTED** [review]
Good work

**Review by amrondonp — APPROVED** [review]
It looks good to me. However, I don't know why the utilities changes are being shown here since I already merged those changes. Could you take a look?

**Comment by amrondonp** [comment]
Alright @crdgonzalezca LGTM, please test it on staging and if it works as expected merge it

**Comment by crdgonzalezca** [comment]
Changes has been tested in staging and everything looks and works good. I am going to merge it.

**Pull request merged**